_scripts/new_workflow_charts.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CHARTS="${CHARTS:-workflow-dev workflow-dev-e2e router-dev}"
+
+for chart in $CHARTS; do
+    new_chart=${chart/-dev/-$WORKFLOW_RELEASE}
+    echo "Creating $new_chart from $chart..."
+
+    # copy the workflow family of charts to their new home
+    cp -R -n "$chart"/ "$new_chart" || \
```

Comment:
It could, but `-n` returns 1 if it can't copy for any reason, which may not be a fatal error. So this allows the script to be re-run if the new chart directories already exist, which I thought might be useful:
```bash
$ WORKFLOW_RELEASE=v2.7.0 _scripts/new_workflow_charts.sh 
Creating workflow-v2.7.0 from workflow-dev...
Failed to copy files. Does workflow-v2.7.0 already exist?
workflow-v2.7.0/tpl/generate_params.toml needs semver values in "dockerTag" fields!
Creating workflow-v2.7.0-e2e from workflow-dev-e2e...
Failed to copy files. Does workflow-v2.7.0-e2e already exist?
workflow-v2.7.0-e2e/tpl/generate_params.toml needs semver values in "dockerTag" fields!
Creating router-v2.7.0 from router-dev...
Failed to copy files. Does router-v2.7.0 already exist?
router-v2.7.0/tpl/generate_params.toml needs semver values in "dockerTag" fields!
```

But I can use an `if` statement if that's more clear.